cmd/cbnt-prov/cmd.go

```diff
@@ -129,6 +129,7 @@ type generateBPMCmd struct {
 	IbbSegsize  uint32               `flag optional name:"ibbsegsize" help:"Value for IBB segment structure"`
 	IbbSegFlag  uint16               `flag optional name:"ibbsegflag" help:"Reducted"`
 	Coreboot    bool                 `flag optional name:"coreboot" help:"Required when BIOS binary file is a coreboot image"`
+	Uefi        bool                 `flag optional name:"uefi" help:"Required when BIOS binary file is a UEFI image"`
```

Comment:
Should the kong 'xox:"coreboot"' flag be used to make them mutually exclusive?
Coreboot will not be adding fit type 7 I suppose and it avoids confusion.